constants.js

```diff
@@ -1,4 +1,4 @@
-const renderTimeout = 30 * 1000;
+const renderTimeout = 60 * 1000;

```

Comment:
The timelimit has been increased as it's been observed that app.webaverse is taking approx 31/32 seconds causing the renderTimout to kick in.

Reply:
That is fine but if it is taking that long we will have problems... The machine should be powerful enough to render this fast.

We should log the test cases where this is problematic.